packages/helm-charts/testnet/README.md

```diff
@@ -1,5 +1,8 @@
 # Testnet Helm Chart
 
+> All helm charts in this package are deprecated and should be removed
```

Comment:
if they are deprecated and not in production anymore I'd just remove them right away

Reply:
Maybe in separate PR so it's more easier to go through files changed / determine if all pipelines work after merging this one 🙏 ? 

I guess there might be some further need to go through rest of documentation and look for usages of `celotool`, `helm-charts` and other unused packages. If `celotool` was an NPM package - it might require going through steps in Notion's "Sunset Process" and looking for usages in community packages (like `celoterminal`).

It's specified as 14th point in "Tasks to improve Devex" doc - so for sure I will get back to it 😁

Reply:
I will do that in another PR to not increase size of this one further more 😉 
Resolving 👍